javacutil/src/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -247,6 +247,18 @@ public static VariableElement findFieldInType(TypeElement type, String name) {
         return null;
     }
 
+    /**
+     * Returns the elements of the fields whose simple names are {@code names} and are declared in
```

Comment:
Or rather I agree that this method should only return non-private fields, but the method below, should return private and non-private fields.